### PR TITLE
Closes #242: Refactor spatial distrib

### LINKED ISSUE #242: Refactor the spatial distribution evaluation
Refactor the existing evaluation for [spatial distributions](https://github.com/mozilla/PRESC/tree/master/presc/spatial_distribution) to conform with the new design for `presc.evaluations`

### PR DESCRIPTION
**Pull request by dzeber**
Porting part of the spatial distribution implementation to the current design.

Leaving tests for later given time constraints.

### CODE REVIEW

**Comment by dzeber** [comment]
@gleonard-m Take a quick look only if you have time given short notice. I haven't added test cases yet for the new code, but it runs as a part of the report.

File: presc/evaluations/spatial_distribution.py [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+from presc.evaluations.utils import is_discrete
+from presc.utils import include_exclude_list
+from presc.configuration import PrescConfig
+from presc import global_config
+
+
+def _pairwise_dist(df1, df2, metrics):
+    """Compute pairwise distances between the rows of two DFs with common columns.
+
```
Comment (gleonard-m) [inline presc/evaluations/spatial_distribution.py:17]: Add Parameters here
       ------------

**Review by gleonard-m — APPROVED** [review]

File: presc/evaluations/spatial_distribution.py (hunk 1 above)
Comment (gleonard-m) [reply presc/evaluations/spatial_distribution.py:17]: Sorry - missed that it was a private function so it won't be included in the docs anyway.

Comment (dzeber) [reply presc/evaluations/spatial_distribution.py:17]: Still, nice to just add the header in as everything else is already formatted.

**Pull request merged**